components/floatingactionbutton/index.css

```diff
@@ -0,0 +1,126 @@
+  --spectrum-floating-action-button-icon-color-key-focus: var(--spectrum-white);
+
+  &--secondary {
+    --spectrum-floating-action-button-background-color: var(--spectrum-background-layer-2-color);
+    --spectrum-floating-action-button-background-color-hover: var(--spectrum-background-layer-2-color);
+    --spectrum-floating-action-button-background-color-down: var(--spectrum-background-layer-2-color);
+    --spectrum-floating-action-button-background-color-key-focus: var(--spectrum-background-layer-2-color);
+    --spectrum-floating-action-button-icon-color: var(--spectrum-neutral-subdued-content-color-default);
+    --spectrum-floating-action-button-icon-color-hover: var(--spectrum-neutral-subdued-content-color-hover);
+    --spectrum-floating-action-button-icon-color-down: var(--spectrum-neutral-subdued-content-color-down);
+    --spectrum-floating-action-button-icon-color-key-focus: var(--spectrum-neutral-subdued-content-color-key-focus);
+  }
+
+}
+
+@media (forced-colors: active) {
+  .spectrum-FloatingActionButton {
+
+    &:after {
+      /* make sure focus-ring renders */
+      forced-color-adjust: none;
+    }
+
+    --highcontrast-floating-action-button-background-color: ButtonText;
+    --highcontrast-floating-action-button-background-color-hover: Highlight;
+    --highcontrast-floating-action-button-background-color-down: Highlight;
+    --highcontrast-floating-action-button-background-color-key-focus: Highlight;
+
+    --highcontrast-floating-action-button-icon-color: ButtonFace;
+    --highcontrast-floating-action-button-icon-color-hover: ButtonFace;
+    --highcontrast-floating-action-button-icon-color-down: ButtonFace;
+    --highcontrast-floating-action-button-icon-color-key-focus: ButtonFace;
+
+  }
+}
+
+.spectrum-FloatingActionButton {
+  cursor: pointer;
+  block-size: var(--mod-floating-action-button-size, var(--spectrum-floating-action-button-size));
+  inline-size: var(--mod-floating-action-button-size, var(--spectrum-floating-action-button-size));
+  border-radius: var(--mod-floating-action-button-border-radius, 50%);
+  padding-inline: var(--mod-floating-action-button-padding, var(--spectrum-floating-action-button-padding));
+  padding-block: var(--mod-floating-action-button-padding, var(--spectrum-floating-action-button-padding));
+  margin-inline-end: var(--mod-floating-action-button-margin, var(--spectrum-floating-action-button-margin));
+  margin-block-end: var(--mod-floating-action-button-margin, var(--spectrum-floating-action-button-margin));
+  border: none;
+  box-shadow: var(--mod-floating-action-button-drop-shadow-x, var(--spectrum-floating-action-button-drop-shadow-x)) var(--mod-floating-action-button-drop-shadow-y, var(--spectrum-floating-action-button-drop-shadow-y)) var(--mod-floating-action-button-drop-shadow-blur, var(--spectrum-floating-action-button-drop-shadow-blur)) var(--mod-floating-action-button-drop-shadow-color, var(--spectrum-floating-action-button-drop-shadow-color));
```

Comment:
--spectrum-floating-action-button-drop-shadow-y is undefined

Reply:
@Rajdeepc This token is defined and resolves to `4px`. Are you using the latest version of tokens?